RSConnectDIOToSnap7/CodeBehind.cs

```diff
@@ -36,614 +36,600 @@
 
 namespace RSConnectDIOToSnap7
 {
-	/// <summary>
-	/// Code-behind class for the RSConnectDIOToSnap7 Smart Component.
-	/// </summary>
-	/// <remarks>
-	/// The code-behind class should be seen as a service provider used by the 
-	/// Smart Component runtime. Only one instance of the code-behind class
-	/// is created, regardless of how many instances there are of the associated
-	/// Smart Component.
-	/// Therefore, the code-behind class should not store any state information.
-	/// Instead, use the SmartComponent.StateCache collection.
-	/// </remarks>
-	public class CodeBehind : SmartComponentCodeBehind
-	{
-		private S7Client client;
-		private bool bCanConnect = false;
-		private bool bPLC_AddrIsValid = false;
-		private bool[] bDI_AddressIsValid = { };
-		private bool[] bDO_AddressIsValid = { };
-
-		/// <summary>
-		/// Called from [!:SmartComponent.InitializeCodeBehind]. 
-		/// </summary>
-		/// <param name="component">Smart Component</param>
-		public override void OnInitialize(SmartComponent component)
-		{
-			///Never Called???

```

Comment:
Did you manage to pass on this method?

Reply:
good question. last time i tested, was before my holidays 3 weeks ago.

Reply:
I think I added this, just for testing purposes.

Reply:
i don't know, the comment is irritating. if it is never called, then the whole function can be removed, otherwise there should be an compiler error.

Reply:
pardon, i mean a runtime error

Reply:
This method is on SmartComponentCodeBehind class.
I've post a question here:
https://forums.robotstudio.com/discussion/11651/smartcomponentcodebehind-oninitialize-method/p1